__tests__/database/schema/ChannelListSchema.test.ts

```diff
@@ -552,12 +553,13 @@ describe('TESTING ChannelListSchema', () => {
         rawJson: expect.any(Object),
         user: null,
         members: null,
-        channelPicture: '',
+        channelPicture:
+          'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
         expiredAt: null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded URL for the default profile picture could be replaced with a constant. This would make it easier to manage and update in the future.
```diff
-        channelPicture: 'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
+        channelPicture: DEFAULT_PROFILE_PICTURE_URL,
```
562:
The function `fromInitAnonymousChatAPI` now accepts an additional parameter, but there is no check for its validity or type. It's good practice to validate all inputs to avoid potential issues down the line.
```javascript
if (typeof type !== 'string') {
  throw new Error('Invalid type parameter');
}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->